build/ProjectReunion-BuildFoundation.yml

```diff
@@ -361,7 +361,7 @@ jobs:
     dependsOn: SignBinariesAndPublishSymbols
     signConfigPattern: 'Microsoft.ProjectReunion*.nupkg'
     useReleaseTag: '$(ProjectReunionFinalRelease)'
-    prereleaseVersionTag: prerelease
+    prereleaseVersionTag: nightly
```

Comment:
VersionTag should be alphabetically sortable -- A<B<C<D...

As this is (the?) earliest in the lifecycle it would preclude naming any tab starting with a-m
Maybe call this "daily" (or "alpha"?)

Reply:
⚠ There's an on-going proposal/discussion around naming, if you push this into main, you're telling the world the official experimental builds are `-nightly` and will have locked us into that naming. @andrewleader are you in sync with @kythant here? This does not seem right.

Reply:
Thanks Rafael, syncing up with Kyaw now to learn more.

Reply:
There's some pressure within the team to get nightly builds running, not just CI to help with integration, testing, and workflow issues. This is independent of the discussion around and release channels, and further churn in package names should probably be expected at this point in the project lifecycle. This name avoid conflation with release channels discussion.

Reply:
Thanks @BenJKuhn. My concern was the merge target (main) here. But perhaps my understanding of the Reunion workflow/branching is incorrect. (It certainly appear to align with typical open source workflows.)